### DISCUSSION TITLE
Consider rating mobile apps

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/tosdr.org#208] [Ideas] [answered]
A lot of services are only accessible through mobile apps, should we do something with that, or stick to rating online services, not software products? See https://groups.google.com/forum/#!topic/tosdr/CN9PEDEaSQg

**Comment by evilcel3ri** [comment]
I believe as more and more apps are very used, they should also have their ToS reviewed :D Are you thinking about services such as Tinder, Snapchat or Instagram that really cannot be used on anything else than a mobile device ?

**Comment by michielbdejong** [comment, marked as answer]
I would like to keep a distinction between (client) app/software and (server) services. Most services do not allow full (API) access from open source clients. So that means we should review the service based on what client app options are offered. But reviewing those actual client apps should be left to separate projects, I think.

Maybe we need a sister project for client app reviews and we can call it "Software License Agreement; Didn't Read" ;)
